docs/pages/partials.mdx

```diff
@@ -0,0 +1,431 @@
+---
+title: Index of Partial Files
```

Comment:
I think we can add this to the "Contribute" section of the docs. It's useful for anyone who wants to write a docs page to refer to.